en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -1,27 +1,27 @@
 # Choreo Connect as a Standalone Gateway
 
-Choreo Connect can be used as a standalone gateway without the API Manager as the control plane. APIs can be deployed via
+Choreo Connect can be used as a standalone Gateway without the API Manager as the control plane. APIs can be deployed via
 APICTL. APICTL can be used to create API projects by including the open API definitions. These API projects can be directly pushed to adapter. The adapter will deploy the API in the enforcer and the router.
 
-When use as a standalone gateway, Choreo Connect can be used with an external Secure Token Service(STS)
-which will provide signed JWTs, and Choreo Connect will validate those JWTs in the incoming requests. When used as a standalone gateway Choreo Connect
+When use as a standalone Gateway, Choreo Connect can be used with an external Secure Token Service(STS)
+which will provide signed JWTs, and Choreo Connect will validate those JWTs in the incoming requests. When used as a standalone Gateway Choreo Connect
 will not apply the subscription validation using the event hub as the control plane is not available. In order to achieve subscription
 validation we need to have subscription data also included in the incoming authentication JWT, which is called as a
 self-contained token.
 
-## Defining gateway specific properties
+## Defining Gateway specific properties
 
-OpenAPI specification doesn't define all required properties for an API gateway. It's meant to be a descriptor of an REST endpoint.
-In this case that would be the OpenAPI definition of backend service or microservice. However, a developer of an API gateway should be able to define,
+OpenAPI specification doesn't define all required properties for an API Gateway. It's meant to be a descriptor of an REST endpoint.
+In this case that would be the OpenAPI definition of backend service or microservice. However, a developer of an API Gateway should be able to define,
```

Comment:
```suggestion
Not all properties required for an API Gateway can be defined in the OpenAPI specification since it is meant to be a descriptor of an REST endpoint such as a backend service or microservice. However, in Choreo Connect a developer  should be able to define,
```